src/stateless/selectbox.vue

```diff
@@ -57,6 +57,8 @@ export default {
         warningText: { type: String, required: false, default: '' },
         options: { type: Array, required: true },
         value: { type: String, required: false, default: '' },
+        getOptions: { type: Function, required: false, default: null },
+        loadAsyncDebounce: { type: Number, default: 0 },
```

Comment:
Does it make sense to make this configurable at this point? We would probably want it always anyway.